### DISCUSSION TITLE
Inventory changer

### DISCUSSION THREAD

**Discussion by shdhevdb** [danielkrupinski/Osiris#3876] [General]
Hi when I add items to my inventory, when i try to exit the osiris menu it doesnt work.
Scenario 1 - I spam the enter key which doesnt work 🥲 
Scenario 2 - I tab out, i hover over the csgo icon in my taskbar, i right click, i try to click csgo.exe but this is what it shows
![image](https://user-images.githubusercontent.com/88394880/201502350-cb4317ac-94b3-44ab-b203-d98768b73d11.png)

Does anyone have a fix for these scenarios? (Ideally the second one but both would be greatly appreciated)

**Comment by Xsintashi** [comment]
1. Menu key is `insert` not `enter`
2. This issue is not Osiris problem, mostly casued by opening csgo out of library from steam.
